bin/upload-schema.js

```diff
@@ -7,39 +7,39 @@ auto pushing schema's tricky. For now this script does the uploads manually.
 */
 
 var aws = require('aws-sdk-promise');
+var fs = require('fs');
 
-var loadConfig = require('taskcluster-base/config');
-var config = loadConfig({
-  defaults: require('../config/defaults'),
-  profile: require('../config/production'),
-  filename: 'docker-worker'
+var base = require('taskcluster-base');
+var config = base.config({
+  files: [`${__dirname}/../config.yml`],
```

Comment:
it appears that the config loading loads config.yml from the current working directory.  So if I'm invoking this outside of the worker directory, then config.yml won't be found.